addons/event/migrations/9.0.0.1/end-migration.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+def create_multiple_attendees_registration_records(env):
+    """On v8, we have one event.registration record = one or several attendees
+    according nb_register field.
+
+    But on v9 we have one event.registration record = one attended, so here
+    we duplicate the existing record for representing the same number of
+    records than attendees.
+
+    Done in end-migration for having all the possible fields in
+    event.registration already loaded.
+    """
+    # Pre-create a column for storing original reference
+    column_name = openupgrade.get_legacy_name('original_registration_id')
```

Comment:
Can you prefix this column with 'openupgrade_' so that it is clear where it came from?

Reply:
It's already done calling `get_legacy_name`...

Reply:
True!